src/timer.rs

```diff
@@ -0,0 +1,27 @@
+use std::time::{Duration, Instant};
+pub struct Timer {
+	period: Duration,
+	init: Instant,
+}
+impl Timer {
+	pub fn time_per_second(times: f32) -> Timer {
+		Timer::new(Duration::from_secs_f32(1.0 / times))
+	}
+	pub fn new(period: Duration) -> Timer {
+		Timer {
+			period,
+			init: Instant::now(),
+		}
+	}
```

Comment:
I'd like to bikeshed these names at some point, but I don't have a concrete suggestion. I just think they should probably be a bit more symmetrical.

Reply:
maybe replace the `new()` with `with_duration` ? Its, a hard one....